build/_build/Build.Utilities.cs

```diff
@@ -168,4 +173,91 @@ partial class Build
                 .SetProcessEnvironmentVariables(envVars));
 
         });
+
+    Target GeneratePackageVersions => _ => _
+       .Description("Regenerate the PackageVersions props and .cs files")
+       .Executes(async () =>
+       {
+           var testDir = Solution.GetProject(Projects.ClrProfilerIntegrationTests).Directory;
+
+           var versionGenerator = new PackageVersionGenerator(RootDirectory, testDir);
+           await versionGenerator.GenerateVersions();
+       });
+
+    Target UpdateVendors => _ => _
+       .Description("Updates the vendored dependency code and dependabot template")
+       .Executes(() =>
+       {
+            var honeypotProject = RootDirectory / "honeypot"  /  "Datadog.Dependabot.Honeypot.csproj";
+            UpdateVendorsTool.UpdateHoneypotProject(honeypotProject);
+
+            var vendorDirectory = Solution.GetProject(Projects.DatadogTrace).Directory / "Vendors";
+            var downloadDirectory = TemporaryDirectory / "Downloads";
+            EnsureCleanDirectory(downloadDirectory);
+            UpdateVendorsTool.UpdateVendors(downloadDirectory, vendorDirectory);
+       });
+
+    Target UpdateIntegrationsJson => _ => _
+       .Description("Update the integrations.json file")
+       .DependsOn(Clean, Restore, CreateRequiredDirectories, CompileManagedSrc, PublishManagedProfiler) // We load the dlls from the output, so need to do a clean build
+       .Before(CopyIntegrationsJson)
+       .Executes(() =>
+        {
+            var assemblies = TracerHomeDirectory
+                            .GlobFiles("**/Datadog.Trace.ClrProfiler.Managed.dll")
+                            .Select(x => x.ToString())
+                            .ToList();
+
+            GenerateIntegrationDefinitions.Run(assemblies, RootDirectory);
+        });
+
+    Target SetVersion => _ => _
+       .Description("Update the version number for the tracer")
+       .Before(Clean, BuildTracerHome)
+       .Requires(() => Version)
+       .Executes(() =>
+        {
+            new SetAllVersions(RootDirectory, Version, IsPrerelease).Run();
+        });
+
+    Target SyncMsiContents => _ => _
```

Comment:
Thanks for removing the word `Synchronize` from `SetVersion`. Can we do the same here? `UpdateMsiContents`?

Reply:
I was going to suggest changing all of the "release" tools to have the same prefix but I thought it could be done in a separate PR:
- `UpdateIntegrationsJson`
- `UpdateVersion`
- `UpdateMsiContents`

Reply:
I agree, these make sense. I updated to use these suggestions in https://github.com/DataDog/dd-trace-dotnet/pull/1605/commits/ce91112ec00649f3bec6a87655d1b3e6653dd049

Reply:
Thanks!